src/QuickMarcEditor/QuickMarcEditorRows/utils.js

```diff
@@ -14,6 +14,16 @@ const INDICATOR_EXEPTION_ROWS = [LEADER_TAG, '001', '002', '003', '004', '005',
 
 export const hasIndicatorException = recordRow => INDICATOR_EXEPTION_ROWS.includes(recordRow.tag);
 
-const ADD_EXCEPTION_ROWS = [LEADER_TAG, '001', '005', '008'];
+const ADD_AND_DELETE_EXCEPTION_ROWS = [LEADER_TAG, '001', '005', '008'];
 
-export const hasAddException = recordRow => ADD_EXCEPTION_ROWS.includes(recordRow.tag);
+export const hasAddException = recordRow => ADD_AND_DELETE_EXCEPTION_ROWS.includes(recordRow.tag);
+
+export const hasDeleteException = recordRow => (
+  ADD_AND_DELETE_EXCEPTION_ROWS.includes(recordRow.tag)
+  || (
+    recordRow.tag === '999'
```

Comment:
it's copy paste....
you again skip comments

Reply:
It was misunderstanding from my side, do you mean that I need to separate part that checks 999ff row?

Reply:
yes